pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -421,6 +422,7 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 								Missing:  []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}},
 							},
 						},
+						PinnedResources: []configv1.GroupVersionKind{{Group: "apps", Version: "v1", Kind: "Deployment"}, {Group: "core", Version: "v1", Kind: "ConfigMap"}},
```

Comment:
I believe it formats this in multiple lines when you add a comma behind the second object.

```suggestion
						PinnedResources: []configv1.GroupVersionKind{
						  {Group: "apps", Version: "v1", Kind: "Deployment"},
						  {Group: "core", Version: "v1", Kind: "ConfigMap"},
						},
```